src/plugins/data_source_management/public/components/data_source_menu/create_data_source_menu.test.tsx

```diff
@@ -71,10 +71,10 @@ describe('create data source menu', () => {
         notifications,
       },
     };
-    const TestComponent = createDataSourceMenu<DataSourceSelectableConfig>(
-      uiSettings,
-      mockDataSourcePluginSetupWithShowLocalCluster
-    );
+    spyOn(utils, 'getApplication').and.returnValue({ id: 'test2' });
+    spyOn(utils, 'getUiSettings').and.returnValue(uiSettings);
+    spyOn(utils, 'getHideLocalCluster').and.returnValue({ enabled: true });
```

Comment:
should we set this to `enabled: false` based on the test case description?

Reply:
@zhongnansu This test itself needs to be fixed since it doesn't change disregards of local cluster is set since it doesn't check the options from popover. We can cut another meta issue for all the tests that we are missing testing, or do you think this needs to block the change?

Reply:
@BionIT agreed that we should fix the text, this is non blocking issue
I think in this PR, the test was modified to remove the check for local cluster existence
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6358/files#diff-ff98db435bf1f4c1bfe7ea1869b5d25b7e731148ce6060333bf37b0dee33219fL78